src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
+/* eslint-disable max-len */
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape : Shape;
+  color : Color;
 
+  getArea() : number;
 }
 
 export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(public color : Color, public a : number,
+    public b : number, public c : number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('One side cannot be equal or longer than sum of both sides');
+    }
+
+    if (a >= b + c) {
```

Comment:
Can't we combine these if statements into one? 

`a >= b + c || b >= a + c || c >= a + b`